src/catalogue/items/catalogueItemsDialog.component.test.tsx

```diff
@@ -155,6 +138,18 @@ describe('Catalogue Items Dialog', () => {
       });
   };
 
+  const modifyManufacturer = async (type: string) => {
+    const manufacturerPopup = screen.getAllByRole('combobox')[0];
+    await user.type(
+      manufacturerPopup,
+      type === 'create'
+        ? 'M{arrowdown}{enter}'
+        : 'Man{arrowdown}{arrowdown}{enter}'
+    );
+    const saveButton = screen.getByRole('button', { name: 'Save' });
+    await user.click(saveButton);
```

Comment:
```suggestion
if (values.manufactuer !== undefined) {
    const manufacturerPopup = screen.getAllByRole('combobox')[0];
    await user.type(
      manufacturerPopup,
      manufacturer
    );
    const saveButton = screen.getByRole('button', { name: 'Save' });
    await user.click(saveButton);
 }
```
Then this can be put into the `modifyValues` and you can use it instead of making a separate call to `modifyManufacturer`

Reply:
I tried doing this initially, but it just does not seem to work. The tests fail for a variety of reasons. Maybe it has to do with the fact I'm using user and not fireEvent